### PR TITLE
addition for including canvas on rails tutorial w/ stimulus

### PR DESCRIPTION
**Pull request by afogel**
Addition for incorporating a tutorial for using canvas w/ stimulus

### CODE REVIEW

File: app/content/pages/use-cases/drawing-on-images.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Drawing on Images
```
Comment (marcoroth) [inline app/content/pages/use-cases/drawing-on-images.html.erb:2]: Could we maybe include "HTML canvas" or just "canvas" in the title as well?

**Review by marcoroth — COMMENTED** [review]
Thanks @afogel! 🙌🏼

File: app/content/pages/use-cases/drawing-on-images.html.erb (hunk 1 above)
Comment (afogel) [reply app/content/pages/use-cases/drawing-on-images.html.erb]: for sure :)

File: app/content/data/blogs.yml [hunk 2]
```diff
@@ -48,3 +48,6 @@
 
 - name: Tony Messias
   url: https://world.hey.com/tonysm
+
+- name: Ariel Fogel
```
Comment (marcoroth) [inline app/content/data/blogs.yml:52]: Also thanks for adding this.

**Pull request merged**